master/buildbot/test/unit/test_stats_service.py

```diff
@@ -313,7 +313,7 @@ def test_build_start_time_capturing(self):
         yield self.end_build_call_consumers()
 
         self.assertEqual(
-            'start-time', self.fake_storage_service.stored_data[0][0].keys()[0])
```

Comment:
This works though:

```
next(iter(iterkeys(self.fake_storage_service.stored_data[0][0]))))
```

Seriously do you want that?  That's pretty ugly, and this is only in test code where
the lists are not large.